src/components/DocumentationTopic/Summary/Availability.vue

```diff
@@ -99,16 +97,23 @@ export default {
 
 <style scoped lang="scss">
 @import 'docc-render/styles/_core.scss';
+$icon-size-default: 20px;
 
 .availability {
   display: flex;
   flex-flow: row wrap;
   gap: 10px;
-  margin-top: 25px;
-}
-
-.badge {
-  margin: 0;
+  margin-top: rem(15px);
+  @include font-styles(body-reduced);
+
+  div:not(:last-child) {
```

Comment:
These selectors are a little messy and hard to follow. I have one idea on a possible way to make it clearer that may or may not work, but it might be worth looking into.

1. Add the same class to every span that you want to have the separator (can be different components)
2. Change the rule to simply be `.that-class + .that-class` (every one preceded by another and apply the left border—no need for using not or last-child, etc)

I may be misunderstanding what this is doing, so feel free to ignore if that's the case.

Reply:
Using sibling selector and apply left border would cause the dividing line to appear on items on a new line when the container wraps. I couldn't find a clean way to not apply border style to first element in each row. Having a dividing line on the right of the last item in a row looks not as broken.
But I added a new class name & cleaned this up a little. Let me know what you think.

Reply:
Understand why you need to do the right border for wrapping 👍

How about something like this maybe?

```diff
diff --git a/src/components/DocumentationTopic/Summary/Availability.vue b/src/components/DocumentationTopic/Summary/Availability.vue
index 29f8156b..e1079bb5 100644
--- a/src/components/DocumentationTopic/Summary/Availability.vue
+++ b/src/components/DocumentationTopic/Summary/Availability.vue
@@ -10,18 +10,18 @@
 
 <template>
   <Section class="availability" role="complementary" :aria-label="$t('sections.availability')">
-    <div
+    <span
       v-for="technology in technologies"
       class="technology"
       :key="technology"
     >
       <TechnologyIcon class="tech-icon" />
-      <span class="availability-info">{{ technology }}</span>
-    </div>
+      <span>{{ technology }}</span>
+    </span>
 
-    <div
+    <span
       v-for="platform in platforms"
-      class="platform availability-info"
+      class="platform"
       :class="changesClassesFor(platform.name)"
       :key="platform.name"
     >
@@ -32,7 +32,7 @@
       />
       <Badge v-if="platform.deprecatedAt" variant="deprecated" />
       <Badge v-else-if="platform.beta" variant="beta" />
-    </div>
+    </span>
   </Section>
 </template>
 
@@ -106,11 +106,14 @@ $icon-size-default: 20px;
   margin-top: rem(15px);
   @include font-styles(body-reduced);
 
-  &:not(:last-child) .availability-info::after {
-    // apply on the span of technology class to ensure same separator height
-    content: " ";
-    border-right: 1px solid;
-    padding-left: 10px;
+  & > * {
+    border-right: 1px solid currentColor;
+    padding-right: 10px;
+
+    &:last-child {
+      border-right: none;
+      padding-right: 0;
+    }
   }
 }
 ```

Reply:
And bonus points if you use a css variable for the `10px` since it would be used for the `gap` and also this padding.

Reply:
I haven't tried Marcus's proposal but it could work. I'd add a small detail: we could revert this change too if we don't use a `content` property https://github.com/apple/swift-docc-render/pull/783/files#diff-61f11105739958fab994f705ecdd503c38e30566ec60f517046ff58c2298d9afL150